### ISSUE TITLE
Missing DLLs

### ISSUE THREAD

**Issue by ProfessorHicman** [Pizzabelly/EasyRP#42]
Program wont launch because its missing DDLs but you haven't said what I need installed (I've put screenshots here, don't judge the Fornite and stuff lol)

I'm probably missing something Python e.g. Python installed but since I don't know what language its in I don't really know.

Would be nice just to say if you need certain things installed (e.g. Python, Node.JS etc)

**Comment by Pizzabelly** [comment]
You are using a broken release from a couple of days ago (my bad :/) download it from the new windows build under releases (v2.0). For reference EasyRP is in c++ and you _shouldnt_ need any other program installed to use it.

**Comment by ProfessorHicman** [comment]
Ok well yes it launches now :3 but theres another issue......

it just gives me this
![image](https://user-images.githubusercontent.com/23481792/42696538-5670050e-86b0-11e8-97a8-83a9f3cb8f92.png)

I've tired 1.0 and 1.0, looked at other examples, recreated the rich presence in the developers area, but still nothing :/

**Comment by Pizzabelly** [comment]
Can you post your config text? Don't worry about clientid its safe to post.

**Comment by ProfessorHicman** [comment]
Okay I don't know how to use Markdown on here so ill just post it normally ;-; (may be same as Discord lol) im also gonna post the files in the folder just in case i'm missing one
I don't think its this but it may be as I tried using two others peoples ones and it still didn't work but I may of derped up XD

[Identifiers]
ClientID=467332896793821184

[State]
State=it's a terrible PC
Details=Someone buy me a new PC
StartTimestamp=
EndTimestamp=

[Images]
LargeImage=potato_pc
LargeImageTooltip=My PC
SmallImage=laughing
SmallImageTooltip=People's Reactions to my PC

![image](https://user-images.githubusercontent.com/23481792/42697172-5208cef4-86b2-11e8-83a1-de259e8395ed.png)

**Comment by Pizzabelly** [comment]
Hmm everything looks good, I would remove the old discord-rpc.dll. It may cause issues with the new exe. If that still doesn't work ill have to wait till I get home to test it.

**Comment by ProfessorHicman** [comment]
Deleting old DLL made 0 difference. Look into it when you have time :3

**Comment by Pizzabelly** [comment]
Alright thanks for the report I'll look into it. But 1 more question, is the discord game registered under the same account you are trying to use the rich presence on?

**Comment by ProfessorHicman** [comment]
Yes it is, I assume that's right?

**Comment by Pizzabelly** [comment]
Yes, that is right. If you could add me ( Pizzabelly#7029 ) to the testers of your app that would be awesome so I can test your config exactaly.

**Comment by Pizzabelly** [comment]
Sweet, thanks for all your help!

**Comment by ProfessorHicman** [comment]
Your welcome

**Comment by Pizzabelly** [comment]
Well your exact settings work for me on linux, so its not a config issue. But I don't have a way to properly test on windows until tomorrow >_>.
![screenshot_20180713_204748](https://user-images.githubusercontent.com/8285260/42719090-709c7f1a-86de-11e8-8293-89dfc33e8825.png)

**Comment by Finnasaurus** [comment]
pizzabelly, this problem can be fixed by having the folder placed in C:\ and not anything deeper.

**Comment by Eric00x** [comment]
It's the problem of **Visual C++ 2015 Redistributable dlls (probably missing or corrupted)**
### Download [**it by clicking this button**](https://www.microsoft.com/en-US/download/details.aspx?id=48145)
If error 0x80070666 occurs, **try downloanding english version of x86** (my pc is x64 and it worked for me)

**Comment by ProfessorHicman** [comment]
Whats weird is that after 2 days it just started working. No idea why, didnt install or modify anything all I had done was play CSGO. Weird, as it wasnt the PC restart as i tried that.

**Comment by Eric00x** [comment]
@ProfessorHicman I guess CS:GO downloaded Visual C++ 2015 Redistributable pack for it, so it works by now.

**Comment by ProfessorHicman** [comment]
Shouldnt off. No update happened.

**Comment by Eric00x** [comment]
Fresh install ? (from two days ago, or more?)

**Comment by ProfessorHicman** [comment]
What of CSGO? No last time it updated was before all of this.
